src/plugins/kdb/db2/libdb2/btree/bt_seq.c

```diff
@@ -239,13 +240,17 @@ __bt_seqset(t, ep, key, flags)
 				return (RET_ERROR);
 
 			/* Check for an empty tree. */
-			if (NEXTINDEX(h) == 0) {
+			if (NEXTINDEX(h) <= 0) {
```

Comment:
NEXTINDEX(h) expands to (h->lower - BTDATAOFF) / sizeof(indx_t).  h->lower is of type indx_t, which is u_int16_t.   I don't think this value can be less than zero.